feature/profile/src/main/java/com/iteneum/profile/presentation/ProfileView.kt

```diff
@@ -0,0 +1,169 @@
+package com.iteneum.profile.presentation
+
+import androidx.compose.foundation.layout.*
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.outlined.LocationOn
+import androidx.compose.material.icons.outlined.Mail
+import androidx.compose.material.icons.outlined.Phone
+import androidx.compose.material3.*
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.sp
+import com.iteneum.designsystem.components.LpEditFloatingActionButton
+import com.iteneum.designsystem.components.LpOutlinedButton
+import com.iteneum.designsystem.theme.LeasePertTheme
+import com.iteneum.profile.R
+
+@Composable
+fun ProfileView(
+
+) {
+    val sizes = LeasePertTheme.sizes
+
+    Column(
+        modifier = Modifier
+            .fillMaxSize()
+            .padding(all = sizes.smallerSize)
+    ) {
+        Row(modifier = Modifier.fillMaxWidth()) {
+            Box(
+                modifier = Modifier.padding(
+                    horizontal = sizes.nothingSize,
+                    vertical = sizes.midSmallSize
+                )
+            ) {
+                /*TODO here will be a nav component*/
+            }
+            Box(
+                modifier = Modifier
+                    .fillMaxWidth(0.75f)
+                    .padding(horizontal = sizes.minorSmallSize, vertical = sizes.minorRegularSize)
+            ) {
+                Text(
+                    text = stringResource(id = R.string.profile),
```

Comment:
I think the name of the view is going to be with the navigation, please check with Irving if this is true? because in the case it is true you don't need all this stuff